### DISCUSSION TITLE
请问有没有相关的触发函数，在Application启动与断开连接时触发

### DISCUSSION THREAD

**Discussion by Viottery** [nonebot/discussions#192] [问答] [answered]
如题，我想在启动时或断开连接时执行一些命令应该用什么函数

**Comment by yanyongyu** [comment, marked as answer]
https://v2.nonebot.dev/docs/next/advanced/runtime-hook#bot-%E8%BF%9E%E6%8E%A5%E5%A4%84%E7%90%86

`driver.on_bot_connect`, `driver.on_bot_disconnect`

**Reply by Viottery** [reply]
谢谢！
